### ISSUE TITLE
rust:1.70.0: no matching manifest for linux/amd64 in the manifest list entries

### ISSUE THREAD

**Issue by yaa110** [docker-library/official-images#14783]
https://github.com/rust-lang/docker-rust/issues/147

**Comment by frederikhors** [comment]
> As noted over there, this is fixed now. 👍

@tianon what happened?

**Comment by frederikhors** [comment]
Yeah what I don't understand is: if there is an issue in the build the image is not published. 1.70 was published but with issues. Is this normal?

**Comment by yosifkit** [comment]
The built images were fine; the only thing missing was an `amd64` based image. The issues users ran into were that all container runtimes on an `amd64` host automatically fallback to `i386` images when the `amd64` one is not in the image index.  The image index for Official Images is created with whatever architectures are ready; it is not gated on any specific architecture being ready.

Current `rust:1.70` image index: https://explore.ggcr.dev/?image=rust%3A1.70%40sha256%3Acf5513bb19a7a59fd271db14b4e71ed5d91df95c4cb1f431ac131231f2e6104a
Previous `rust:1.70` image index: https://explore.ggcr.dev/?image=rust%3A1.70%40sha256%3A79e811f25051df6f9fe20e24ffd8102b366d2d38dd0f4eec2bef7ddb84b23f5f

---

Perhaps a walkthrough of the process is helpful: https://github.com/docker-library/faq/tree/da53abbaf0ed592989e5f8b50adcbd062582033d#an-images-source-changed-in-git-now-what
